src/olympia/devhub/templates/devhub/includes/addon_details.html

```diff
@@ -99,24 +128,10 @@
       {{ addon.last_updated|datetime(_('%%b %%e, %%Y')) }}
   </li>
 {% endif %}
-{% if addon.has_listed_versions() %}
-  {% set latest_listed_version=addon.find_latest_version(channel=amo.RELEASE_CHANNEL_LISTED) %}
-  {% if latest_listed_version and latest_listed_version != addon.current_version %}
-    <li>
-      <strong>{{ _('Next Version:') }}</strong>
-      {{ link_if_listed_else_text(latest_listed_version, latest_listed_version.version) }}
-      <span class="tip tooltip" title="{{ _('This is the newest uploaded version, however it isn’t live on the site yet.') }}">?</span>
-    </li>
-  {% endif %}
-  {% with position = get_position(addon) %}
-    {% if position and position.pos and position.total %}
-      <li class="queue-position" title="{{ _('Queues are not reviewed strictly in order') }}">
-        <strong>{{ _('Queue Position:') }}</strong>
-        {% trans position=position.pos|numberfmt,
-                 total=position.total|numberfmt %}
-          {{ position }} of {{ total }}
-        {% endtrans %}
-      </li>
-    {% endif %}
-  {% endwith %}
+{% if addon.type == amo.ADDON_EXTENSION and amo.FIREFOX in addon.compatible_apps %}
```

Comment:
Note: there is a slight difference in the way `Version` and `Addon` handle `compatible_apps`, when it comes to search providers and Personas. So maybe don't shave that yak yet...